saleor/graphql/order/tests/test_order.py

```diff
@@ -5830,6 +5831,94 @@ def test_orders_query_with_filter_search_by_id_with_hash(
     assert content["data"]["orders"]["totalCount"] == 1
 
 
+def test_orders_query_with_filter_search_by_product_sku_with_allocation(
```

Comment:
It doesn't matter if orders have allocation or not, filters will work in the same way. No need to include this in the name of tests. Now, test name suggests that there is an relation between search by sku and allocation